egov-dss-dashboards/dashboard-analytics/MasterDashboardConfig.json

```diff
@@ -10046,6 +10046,45 @@
           ]
         }
       ]
+    },
+    {
```

Comment:
@ajay-egov As this config will be used only for the new health dashboard, we can clean up the existing config data from this file and include only the new health dashboards

Reply:
@neel-tw , removed the existing config data